packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_cookies.spec.ts

```diff
@@ -0,0 +1,39 @@
+// @ts-ignore / session support is needed for visiting about:blank between tests
+// FIXME: hangs
```

Comment:
any idea why this hangs or what is causing it?

Reply:
No, I haven't investigated the cause. Right now it hangs calling `getCookies()` without any errors or anything in the console log.